### ISSUE TITLE
Azure webhook function: Exception-handling

### ISSUE THREAD

**Issue by jansoren** [Azure/azure-webjobs-sdk#1377]
I'm working on exception-handling in my Azure webhook function.
I would like to be able to catch all exceptions in a filter and return a default response (something like `req.CreateResponse(HttpStatusCode.InternalServerError);`) from my webhook.

Is such a task ment for `FunctionExceptionFilterAttribute`, and how to i get the request/response from the context? Or is it another way to do this in a webhook azure function?

**Comment by paulbatum** [comment]
Have you tried using exception filters as discussed [here](https://github.com/Azure/azure-webjobs-sdk/wiki/Function-Filters#filter-types)?

**Comment by jansoren** [comment]
Yes, that was what i reffered to in my initial comment. There is nothing there that makes it possible to access the request/response object :-/

**Comment by paulbatum** [comment]
@mathewc is this possible?

**Comment by jansoren** [comment]
@mathewc is this possible?

**Comment by patricknolan** [comment]
I agree that without the ability to return a certain result object, such as BadRequestObjectResult based on the exception, the FunctionExceptionFilterAttribute isn't overly useful.
